htdocs/expedition/class/expedition.class.php

```diff
@@ -238,7 +245,7 @@ function create($user)
 		$sql.= ", ".(!empty($this->model_pdf)?"'".$this->db->escape($this->model_pdf)."'":"null");
 		$sql.= ")";
 
-		dol_syslog(get_class($this)."::create", LOG_DEBUG);
+		dol_syslog(get_class($this)."::create sql=".$sql, LOG_DEBUG);
```

Comment:
The thing that is being logged is the query. That line is used to recognize which function called the query. Like a separator.